tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -426,7 +429,23 @@ def test_montecarlo_line_scatter(packet_params, expected_params, packet, model,
     assert_almost_equal(packet.tau_event, expected_params['tau_event'])
     assert_almost_equal(packet.next_line_id, expected_params['next_line_id'])
 
+"""
+Simple Tests:
+----------------
+These test check very simple pices of code still work.
+"""
+
+@pytest.mark.parametrize(
+    ['packet_params', 'line_idx', 'expected'],
+    [({'energy':0.0}, 0, 0),
+     ({'energy':1.0}, 1, 1),
+     ({'energy':0.5}, 2, 1.5)]
+)
```

Comment:
I think the line you link to is quite a bit harder to read... is it very much faster to declare in tuples, or it is the immutability you advocate for?